.eslintrc.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  extends: ['@favware/eslint-config-react-app', 'plugin:prettier/recommended'],
+  overrides: [
+    {
+      files: ['./tsdx.config.js'],
+      rules: {
+        '@typescript-eslint/no-var-requires': 0
+      }
+    }
+  ]
+};
```

Comment:
why JS tho... JSON was alright

Reply:
Covered in df46295605c9b177909666a56225216ca6c7e1b0